types/react-big-calendar/index.d.ts

```diff
@@ -284,31 +305,50 @@ export interface Messages {
     noEventsInRange?: string | undefined;
 }
 
+/**
+ * @resourceId For "TimeGrid" views
+ * @bounds For "select" action
+ * @box For "click" or "doubleClick" actions
+ */
 export interface SlotInfo {
-  start: stringOrDate;
-  end: stringOrDate;
-  slots: Date[] | string[];
-  action: 'select' | 'click' | 'doubleClick';
+    start: Date;
+    end: Date;
+    slots: Date[];
+    action: 'select' | 'click' | 'doubleClick';
+    resourceId?: number | string;
+    bounds?: {
+        x: number;
+        y: number;
+        top: number;
+        bottom: number;
+        left: number;
+        right: number;
+    };
+    box?: {
+        x: number;
+        y: number;
+        clientX: number;
+        clientY: number;
+    };
 }
 
-export type Culture = string;
```

Comment:
I suspect that this is a breaking change because it is exported; what is the rationale for eliminating this type?

Reply:
I would rather question what is the rationale for it to exist

Reply:
I don't have context for why it was written this way, but my role in reviewing this is to try and prevent unnecessary breaks for people who are using the types now, and deleting an exported type is one that I suspect will cause issues for people who have imported it.

Reply:
I can see how removing an exported member is BC. I removed it because i dont see the value of it while it's just pointing to a primitive. Also there are mixed assignaments of culture, some pointiring to `Culture` type while others to [string | undefined](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-big-calendar/index.d.ts#L358). Can we just bump it?

Reply:
You're right that it is inconsistent. I am curious what the owners think, though I think only @michalak111 gave a review.

Reply:
I'd prefer to leave the `Culture` type and in places [string | undefined](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-big-calendar/index.d.ts#L358) update them to `Culture | undefined`